src/main/resources/org/jenkinsci/plugins/authorizeproject/form/dropdownDescriptorSelector.jelly

```diff
@@ -66,7 +66,7 @@ THE SOFTWARE.
       <f:dropdownListBlock value="${loop.index}" title="${descriptor.displayName}"
                            selected="${current.descriptor==descriptor || (current==null and descriptor==attrs.default)}"
                            staplerClass="${descriptor.clazz.name}"
-                           lazy="${attrs.lazy?attrs.lazy+',':''}descriptor,it">
+                           lazy="${(attrs.capture != null and attrs.capture != '')?attrs.capture+',':''}descriptor,it">
```

Comment:
Expressions in jelly seem JEXL: http://commons.apache.org/proper/commons-jexl/reference/syntax.html

Anyway, the inconsistent behaviors of ternary conditional operators and elvis operators aren't defined even in JEXL, and might be a bug.